lib/lightning/auditing.ex

```diff
@@ -12,7 +12,16 @@ defmodule Lightning.Auditing do
     from(a in Audit,
       left_join: u in User,
       on: [id: a.actor_id],
-      select_merge: %{actor: u},
+      select_merge: %{
```

Comment:
The next iteration of this query is likely to be a Union  - some experimentation with a union and a select merge that maps to a Struct produced issues. So, for now, I am going with a dumber version that will hopefully be more acceptable once the union is implemented.